test/clj/clojure/next_test.clj

```diff
@@ -60,3 +60,10 @@
     (is
       (nil? (binding [*assert* false] (eval '(assert false)))))))
 
+(deftest conj-test
+  (testing "conj with no arguments returns an empty vector"
+    (is (= [] (conj))))
+
+  (testing "conj with one argument returns the argument"
+    (is (= '(1 2 3) (conj '(1 2 3))))))
```

Comment:
I'd like to see the `coll x` and `coll x & xs` cases explicitly tested here. The test suite is not just to verify behavior, but to document the usage.